acquia/blt
# Returning empty array instead of all filesets when no fileset is given.

Fixes #3133 
--------

Changes proposed:
---------
Returning empty array instead of all filesets when no fileset is given so we can skip linting when necessary.

Steps to verify the solution:
-----------

1. Set validate.twig.filesets to false on blt/blt.yml
2. Run blt tests:twig:lint:all

Command should do nothing and return success.
